src/Controls/src/Core/Shell/BaseShellItem.cs

```diff
@@ -537,6 +537,16 @@ internal static DataTemplate CreateDefaultFlyoutItemCell(string textBinding, str
 				return grid;
 			});
 		}
+
+#if NETSTANDARD
+		sealed class OperatingSystem
```

Comment:
Is this the right place for it? Do we need this in more places?

Reply:
Maybe? The problem I ran into putting this into a more general place is that you start to get errors about "OperatingSystem" is defined in multiple places. Unfortunately, NS has the `System.OperatingSystem` type it just doesn't have anything useful for us. 

Another attempt I had was to just call it `OperatingSystemNS` and then add a special using at the top?

Not sure the cleverest way to make this reusable.

Maybe we add our own internal static? `MAUIOS` that we can use generally?